pmd-cli/src/main/java/net/sourceforge/pmd/cli/commands/internal/AbstractPmdSubcommand.java

```diff
@@ -26,6 +29,8 @@ public abstract class AbstractPmdSubcommand implements Callable<Integer> {
 
     @Override
     public final Integer call() throws Exception {
+        System.err.println(PmdBanner.loadBanner().stream().collect(Collectors.joining(System.lineSeparator())));
+        System.err.println(PMDVersion.getFullVersionName());
```

Comment:
I'm not sure I love this… it's noisy for a CLI that is meant to be clean. I'm ok on having it on `--version`, but for every single run…

Reply:
Ok, let's keep it then only for `--version` for now.

I wasn't entirely aware, that we have an implicit "--quite" mode :).
I think eventually, we should have something like `--show-version` which prints the version on _every_ run of PMD. Often problems arise, because you assume you are using a specific version of a tool, but actually you don't. Then it's helpful to just scroll up the log to verify/double check your assumption. Especially for CI builds, where you might only have access to the log.

I'll update this PR the next days.

Reply:
Done.